frontend/src/components/PaginatedCardGrid.jsx

```
@@ -0,0 +1,173 @@
+        alignItems: 'center',
+      }}>
+        <div style={{
+          color: '#ff3b30',
+          fontSize: '24px',
+          fontWeight: '500',
+          display: 'flex',
+          alignItems: 'center',
+          gap: '8px',
+        }}>
+          <span>LiveFeed</span>
+          <span style={{ color: '#666' }}>{currentTime} // {currentDate} // {data.length} Makers</span>
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)